script.js

```diff
@@ -318,3 +319,37 @@ form.addEventListener("submit", (e) => {
       "Please, use only lowercase letter for your email!";
   }
 });
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('userInfo')) {
+    const userInfo = JSON.parse(localStorage.getItem('userInfo'));
+    usernameInput.value = userInfo.name;
+    emailInput.value = userInfo.email;
+    messageText.value = userInfo.message;
+  } else {
+    localStorage.setItem('userInfo', userData);
+  }
+});
+
+function saveData(key, value) {
+  if (localStorage.getItem('userInfo')) {
+    const oldData = JSON.parse(localStorage.getItem('userInfo'));
+    const newData = {
+      ...oldData,
+      [key]: value
+    };
+    localStorage.setItem('userInfo', JSON.stringify(newData));
+  } else {
+    localStorage.setItem('userInfo', userData);
+  }
+}
```

Comment:
* Currently changes made on the input fields are not stored in local storage

Reply:
ok corrected it